### DISCUSSION TITLE
cargo clippy possibly in settings.toml file

### DISCUSSION THREAD

**Discussion by charleschege** [lapce/lapce#1218] [General]
How do I set cargo clippy to run instead of cargo check? If it is in the settings file, What are the contents of the settings.toml file? Can't find the documentation

**Comment by dzhou121** [comment]
You can put these in the settings.toml

```
[lapce-rust]
checkOnSave.command = "clippy"
```

**Reply by charleschege** [reply]
Thanks. 
I suggest there be some documentation about this in the website docs

**Reply by dzhou121** [reply]
It will be exposed to be plugin settings in the future.
